em_workflows/sem_tomo/flow.py

```diff
@@ -207,6 +209,42 @@ def gen_keyimg_small(fp_in: FilePath) -> Dict:
     return keyimg_asset
 
 
+@task
+def gen_zarr(fp_in: FilePath) -> None:
+    file_path = fp_in
+    # fallback mrc file
+    input_file = file_path.fp_in.as_posix()
+
+    base_mrc = file_path.gen_output_fp(output_ext=".mrc", out_fname="adjusted.mrc")
+    if base_mrc.is_file():
+        input_file = base_mrc.as_posix()
+
+    ng.bioformats_gen_zarr(
+        file_path=file_path,
+        input_fname=input_file,
+        depth=FIBSEM_DEPTH,
+        width=FIBSEM_WIDTH,
+        height=FIBSEM_HEIGHT,
+        resolutions=1,
+    )
+    ng.zarr_build_multiscales(fp_in)
+
+
+@task
+def gen_ng_metadata(fp_in: FilePath) -> Dict:
+    file_path = fp_in
+    output_zarr = f"{file_path.working_dir}/{file_path.base}.zarr"
```

Comment:
should be Path(output_zarr) - or create Path object when passing to gen_asset on 242